source/core_rrv/packages/core_rrv_ips_pkg.vh

```diff
@@ -52,6 +52,79 @@ typedef struct packed {
     logic       kbd_scanf_en;
 } t_kbd_cr;
 
+// Ascii code to scan code. // TODO include special characters
```

Comment:
I  think this is the Type of content that we don't really need in the RTL package. we should probably have a DV package for things that are used only on DV (design verification - AKA TB)